phase2/addon/lib/controller.js

```diff
@@ -655,6 +656,12 @@ const deliverer = {
 
     let rejectDelivery = false;
 
+    if (dh.buttonInited() && (!dh.buttonPlacement() || dh.buttonPlacement() == 'addon-bar')){
```

Comment:
rejects delivery.
if dh.buttonPlacement() is null, it means that it's been moved to the customization panel